types/chart.js/index.d.ts

```diff
@@ -663,6 +664,7 @@ declare namespace Chart {
     }
 
     interface RadialLinearScale extends LinearScale {
+        display?: boolean;
```

Comment:
RadialLinearScale extends LinearScale which extends ChartScales which already define the ```display``` property.